website/static/twitter/credentials.py

```diff
@@ -0,0 +1,14 @@
+
+
+# Create variables for each key, secret, token
```

Comment:
Ok! Yes, I'm sorry, I forgot to remove that twitter folder. It's not necessary. Thanks for catching that!